self-core-impl/src/main/java/com/selfxdsd/core/GitlabOrganizationRepos.java

```diff
@@ -0,0 +1,94 @@
+package com.selfxdsd.core;
+
+import com.selfxdsd.api.Repo;
+import com.selfxdsd.api.Repos;
+import com.selfxdsd.api.User;
+import com.selfxdsd.api.storage.Storage;
+
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+import javax.json.JsonValue;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.Iterator;
+
+/**
+ * A Gitlab Provider Organization Repos.
+ *
+ * @author criske
+ * @version $Id$
+ * @since 0.0.9
+ */
+final class GitlabOrganizationRepos implements Repos {
+
+    /**
+     * Organization Repos URI owned by current User.
+     */
+    private final URI uri;
+
+    /**
+     * Current authenticated User.
+     */
+    private final User owner;
+
+    /**
+     * Gitlab's JSON Resources.
+     */
+    private final JsonResources resources;
+
+    /**
+     * Storage used by Organization Repo.
+     */
+    private final Storage storage;
+
+    /**
+     * Ctor.
+     * @param organizationId Organization Id.
+     * @param owner Current authenticated User.
+     * @param resources Gitlab's JSON Resources.
+     * @param storage Storage used by Organization Repo.
+     */
+    GitlabOrganizationRepos(final String organizationId,
+                            final User owner,
+                            final JsonResources resources,
+                            final Storage storage) {
+        this.uri = URI.create("https://gitlab.com/api/v4/groups/"
+            + organizationId + "/projects?simple=true&owned=true");
```

Comment:
@criske the URI should be constructed on the fly, starting from ``GitLab`` and going down.
See how Github is implemented :D 

Or is there a special reason why you did it like this?

Reply:
Is easier to test. For Github makes sense cause there we dealing with org repos uri extracted from json.
https://github.com/self-xdsd/self-core/blob/dd2648290b4baad63189299c60deb8f2e33daac2/self-core-impl/src/test/java/com/selfxdsd/core/GithubOrganizationTestCase.java#L53
But here we only need the org `id` and with this aproach I'll avoid testing the uri twice: in` GitlabOrganizationTestCase` and in `GithubOrganizationReposTestCase`; But only once see: `GitlabOrganizationReposTestCase#fetchesAndFilterOk` in PR